libraries/3rdparty/CMakeLists.txt

```diff
@@ -46,3 +46,33 @@ set_target_properties(tinycbor PROPERTIES
 target_include_directories(tinycbor PUBLIC
     "${ROOT_DIR}/libraries/3rdparty/tinycbor/src"
 )
+
+file(GLOB MBEDTLS_FILES CONFIGURE_DEPENDS "${ROOT_DIR}/libraries/3rdparty/mbedtls/library/*.c")
+
+add_library(mbedtls
+    ${MBEDTLS_FILES}
+)
+
+set_target_properties(mbedtls PROPERTIES
+    ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib
+    POSITION_INDEPENDENT_CODE ON
+    C_STANDARD 99
+)
```

Comment:
Might I suggest
```suggestion
set_target_properties(mbedtls PROPERTIES
    ARCHIVE_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/lib
    POSITION_INDEPENDENT_CODE ON
)

# Use C99 for mbedtls as v2.26.0 is incompatible with C90
if(CMAKE_C_STANDARD LESS 99)
    set_target_properties(mbedtls PROPERTIES C_STANDARD 99)
endif()
```
to allow this to be compiled with a later standard if manually specified?